lib/htmldump.js

```diff
@@ -130,6 +131,7 @@ function Dumper (articleChunkStream, options) {
     this.articles = [];
     this.waiters = [];
     this.done = false;
+    this.contentHost = 'http://' + options.prefix + '/api/rest_v1'
```

Comment:
Good point, we still need to set the `Host` header. Perhaps we could then change the meaning of `prefix` to be what is now `contentHost` and keep `host` as a command-line arg? So we'd have both `prefix` and `host`, but `prefix` would be the **URI prefix** to use and `host` would be used for the header and the domain in the requests. The latter could be optional, so that if it's not specified, we can simply `url.parse()` it out of `prefix`.